addon/services/i18n.js

```diff
@@ -153,9 +162,10 @@ const I18nService = Ember.Service.extend({
     }).then(() => {
       return this._runPostInitActions(oldLang);
     }).then(() => {
-      return Ember.RSVP.resolve(lang);
+      return resolve(lang);
     }).catch(reason => {
-      Ember.Logger.warn(`A promise in the locale change path rejected with reason: ${reason}`);
+      // eslint-disable-next-line no-console
+      console.warn(`A promise in the locale change path rejected with reason: ${reason}`);
```

Comment:
The RFC for removing `Ember.Logger` is in final comment period, and there's already work being done to remove it (https://github.com/emberjs/ember.js/issues/16231).